src/logic/is_empty.lean

```diff
@@ -83,14 +83,16 @@ instance : subsingleton α := ⟨is_empty_elim⟩
 
 end is_empty
 
+variables (α)
+
 @[simp] lemma not_nonempty_iff : ¬ nonempty α ↔ is_empty α :=
 ⟨λ h, ⟨λ x, h ⟨x⟩⟩, λ h1 h2, h2.elim h1.elim⟩
 
 @[simp] lemma not_is_empty_iff : ¬ is_empty α ↔ nonempty α :=
-not_iff_comm.mp not_nonempty_iff
+not_iff_comm.mp (not_nonempty_iff α)
```

Comment:
I'm not convinced adding the explicit argument is useful for these `iff`s (but it's definitely useful for `is_empty_or_nonempty` below).

Reply:
Ok, I'll revert those.